config/manager/manager.yaml

```diff
@@ -47,3 +47,11 @@ spec:
       tolerations:
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
+---
+kind: ConfigMap 
+apiVersion: v1 
+metadata:
+  name: fast-track-cm
+  namespace: system
+data:	
+  fastTrack: "false"
```

Comment:
`fastTrack` defaults to true in ironic-image, why false here?

Reply:
Currently capm3 powers off the BMH after deprovisioning. Not to change the current behaviour we decided that default value to be "false".  
Default value for AutomatedCleaningMode is "metadata".

Reply:
It doesn't only affect deprovisioning. It also means that you will have 2 additional reboots: after inspection and after initial cleaning (before deployment). This makes provisioning significantly longer.

If CAPM3 powers off BMH itself, the fast track value will not matter: Ironic will obey.

Reply:
fastTrack variable is not tied the one with in Ironic. This fastTrack variable is just to control whether CAPM3 should turn off the BMH after deprovisioning or not.

Reply:
@dtantsur, Not to confuse Ironic fast track variable with the one introduced in this patch, I have renamed to Capm3FastTrack